Sources/Apodini/Configurations/HTTPConfiguration.swift

```diff
@@ -53,7 +53,7 @@ public final class HTTPConfiguration: Configuration {
                 let port = components.last.flatMap { Int($0) }
                 self.address = .hostname(hostname, port: port)
             case let (hostname, port, .none, .none):
-                self.address = .hostname(hostname ?? Defaults.hostname, port: port ?? Defaults.port)
+                self.address = .hostname(hostname, port: port)
```

Comment:
Looks like we covered the absence already in the first case (so no need to check?). I changed the first case to use the default values